src/tests/utils/convert.js

```diff
@@ -1,20 +1,31 @@
 require('should');
+const requireFromString = require('require-from-string');
 const convert = require('../../utils/convert');
 const definitions = {
   basic: require('./definitions/basic.json'),
+  basicScripting: require('./definitions/basic-scripting.json'),
```

Comment:
`basic-scripting` is the same as `basic.json` except that it has a `get_connection_info` method in scripting.